spec/factories/members.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :member do
+    name 'Chase'
+    github_login 'chaserx'
+    github_id 101010
+    avatar_url 'https://avatars2.githubusercontent.com/u/7292?v=3&s=460'
+    profile_url 'https://github.com/chaserx'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.